src/components/GlobalStyle.jsx

```diff
@@ -0,0 +1,78 @@
+import { Global, css } from '@emotion/react';
+
+const GlobalStyle = () => (
+  <Global
+    styles={css`
+      @font-face {
+        font-family: 'Pretendard-Thin';
+        src: url('https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Thin.woff')
+          format('woff');
+        font-weight: 100;
+        font-style: normal;
+      }
+      @font-face {
+        font-family: 'Pretendard-ExtraLight';
+        src: url('https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-ExtraLight.woff')
+          format('woff');
+        font-weight: 200;
+        font-style: normal;
+      }
```

Comment:
폰트의 종류가 같다면 `font-family`는 하나의 이름으로 설정해 주시면 됩니다!
```suggestion
      @font-face {
        font-family: 'Pretendard';
        src: url('https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Thin.woff')
          format('woff');
        font-weight: 100;
        font-style: normal;
      }
      @font-face {
        font-family: 'Pretendard';
        src: url('https://fastly.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-ExtraLight.woff')
          format('woff');
        font-weight: 200;
        font-style: normal;
      }
```